scripts/Muon/GUI/Common/home_instrument_widget/home_instrument_widget_view.py

```diff
@@ -14,6 +14,11 @@
 from Muon.GUI.Common.utilities.run_string_utils import valid_float_regex
 from Muon.GUI.Common.message_box import warning
 
+DEADTIME_DATA_FILE = 0
```

Comment:
I won't hold up merging this for the release but I'd recommend an [`Enum`](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/mantid/py3compat/enum.py) to keep the declarations together and protect against unknown values. Maybe something to do in maintenance?

Reply:
@LolloB could you make an issue for this and do the fix for master once you have no release tasks left